services/apollo/Dockerfile

```diff
@@ -1,15 +1,30 @@
-FROM node:10.15.0
-ARG VERSION
-ENV PREFECT_SERVER_VERSION=${VERSION}
+ARG NODE_VERSION=${NODE_VERSION:12}
```

Comment:
We'll need to do some heavy QA on our side to make sure this upgrade is non-breaking, so we should stick to `10.15.0` for now and open an issue for upgrading to `12`

Reply:
@cicdw There's currently 3 point releases for 10.15, does it have to be specifically 10.15.0 or would 10.15.3 work?